docs/_pages/commercial-support/_pricing-table.html

```diff
@@ -0,0 +1,498 @@
+				</div>
+			</div>
+			<div class="cell-cnt col-md-2">
+				<div class="cell">
+					<span class="sla">1 biz day</span>
+				</div>
+			</div>
+		</div>
+		<div class="single-row">
+			<div class="cell-cnt col-md-6">
+				<div class="description">
+					<div class="sla-badge">SLA</div>
+					<span>Code reviews</span>
+				</div>
+			</div>
+			<div class="cell-cnt col-md-2">
+				<div class="cell">
+					<span class="sla">N/A</span>
+				</div>
+			</div>
+			<div class="cell-cnt col-md-2">
+				<div class="cell">
+					<span class="sla">N/A</span>
+				</div>
+			</div>
+			<div class="cell-cnt col-md-2">
+				<div class="cell">
+					<span class="sla">1 biz day</span>
+				</div>
+			</div>
+		</div>
+		<div class="single-row">
+			<div class="cell-cnt col-md-6">
+				<div class="description">
+					<div class="sla-badge">SLA</div>
+					<span>Prioritized bug fixes</span>
+				</div>
+			</div>
+			<div class="cell-cnt col-md-2">
+				<div class="cell">
+					<span class="sla">N/A</span>
+				</div>
+			</div>
+			<div class="cell-cnt col-md-2">
+				<div class="cell">
+					<span class="sla">N/A</span>
+				</div>
+			</div>
+			<div class="cell-cnt col-md-2">
+				<div class="cell">
+					<span class="sla">2 biz days</span>
+				</div>
+			</div>
+		</div>
+		<div class="single-row">
+			<div class="cell-cnt col-md-6">
+				<div class="description">
+					<div class="sla-badge">SLA</div>
+					<span>Prioritized features *</span>
+				</div>
+			</div>
+			<div class="cell-cnt col-md-2">
+				<div class="cell">
+					<span class="sla">N/A</span>
+				</div>
+			</div>
+			<div class="cell-cnt col-md-2">
+				<div class="cell">
+					<span class="sla">N/A</span>
+				</div>
+			</div>
+			<div class="cell-cnt col-md-2">
+				<div class="cell">
+					<span class="sla">5 biz days</span>
+				</div>
+			</div>
+		</div>
+		<div class="single-row">
+			<div class="cell-cnt borderless-cell col-md-6">
+			</div>
+			<div class="cell-cnt borderless-cell col-md-2">
+				<div class="cell footer-cell">
+					<span class="footer-price">Free</span>
+				</div>
+			</div>
+			<div class="cell-cnt borderless-cell col-md-2">
+				<div class="cell footer-cell">
+					<span class="footer-price">$35k / year </span>
+				</div>
+			</div>
+			<div class="cell-cnt borderless-cell col-md-2">
+				<div class="cell footer-cell">
+					<span class="footer-price">Let's talk </span>
+				</div>
+			</div>
+		</div>
+		<div class="single-row">
+			<div class="cell-cnt borderless-cell col-md-6">
+			</div>
+			<div class="col-md-2 borderless-cell contact-cell">
+				<div class="footer-cell-contact">
+					<a href="https://gruntwork.io" target="_blank" class="btn btn-secondary btn-contact-pricing btn-sm">Start Now</a>
+				</div>
+			</div>
+			<div class="col-md-2 borderless-cell contact-cell">
+				<div class="footer-cell-contact">
+					<a href="/contact?plan=pro" target="_blank" class="btn btn-primary btn-contact-pricing btn-sm">Contact Sales</a>
```

Comment:
For the "Start Now" and "Contact Sales" buttons, could you:

1. Put more vertical space above them (e.g., `3rem` instead of `1rem`), so they aren't so close to the pricing columns.
1. Add more padding within the buttons. Or, more accurately, make the buttons a bit taller, tiny bit wider, and therefore, make the text in the buttons feel less crammed.

Reply:
Also, why `target="_blank"` on these buttons?